### PR TITLE
[Example] Support multiple inputs/outputs for Model Importer and do code refactor for SSD_Mobilenet

### LINKED ISSUE #420: [Example] Code refactor for SSD_Mobilenet example

### PR DESCRIPTION
**Pull request by Wenzhao-Xiang**
Fix #420

### CODE REVIEW

File: examples/util/tflite/TFliteModelImporter.js [hunk 1]
```diff
@@ -38,9 +38,13 @@ class TFliteModelImporter {
     this._execution = await this._compilation.createExecution();
   }
 
-  async compute(inputTensor, outputTensor) {
-    this._execution.setInput(0, inputTensor);
-    this._execution.setOutput(0, outputTensor);
+  async compute(inputTensors, outputTensors) {
```
Comment (Wenzhao-Xiang) [inline examples/util/tflite/TFliteModelImporter.js]: Here support the mutiple inputs/outputs. And all the examples that use TFliteImporter also do some changes for it.

**Comment by Wenzhao-Xiang** [comment]
Also did changes to OnnxImporter and Deeplab, @pinzhenx please help take a look.

File: examples/object_detection/model/coco_labels_list.txt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+cow
+elephant
+bear
+zebra
+giraffe
+???
+backpack
+umbrella
+???
+???
```
Comment (ibelem) [inline examples/object_detection/model/coco_labels_list.txt]: I am not sure if so many ??? here are correct, but is is the same as the upstream file https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/examples/android/app/src/main/assets/coco_labels_list.txt

Comment (Wenzhao-Xiang) [reply examples/object_detection/model/coco_labels_list.txt]: Please look at here:
https://stackoverflow.com/questions/50665110/tensorflow-models-uses-coco-90-class-ids-although-coco-has-only-80-categories

I guess the dataset for this object detection model is the 2014 released version(only contains 80 object categories).

**Comment by Wenzhao-Xiang** [comment]
@ibelem Please help take a look at this PR.

**Comment by ibelem** [comment]
LGTM, thank!

**Pull request merged**